### PR TITLE
Feature/nypl

### PR DESCRIPTION
**Pull request by anarchivist**

### CODE REVIEW

File: lib/shimmy/shims/nypl_hires_maps.rb [hunk 1]
```diff
@@ -0,0 +1,56 @@
+require 'nypl_repo'
+require 'iiif/presentation'
+require 'pry'
+
+# Before you can use this, you need to set your Flickr API credentials:
+#
+# >>> FlickRaw.api_key = 'YOUR_API_KEY'
+# >>> FlickRaw.shared_secret = 'YOUR_SHARED_SECRET'
```
Comment (mejackreed) [inline lib/shimmy/shims/nypl_hires_maps.rb]: Should we put info about nypl api stuff here instead?

Comment (anarchivist) [inline lib/shimmy/shims/nypl_hires_maps.rb]: Force pushed. This is fixed in b3571fe

**Comment by anarchivist** [comment]
OK, I got this  working for both highres map images, and more generally, for any available image. If you look at the comments you'll see that sometimes higher res versions are available for download... but that info isn't available in the public API.

**Comment by mejackreed** [comment]
:+1:

**Pull request merged**